src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -233,9 +234,11 @@ export const CourseRunEnrollment: React.FC<CourseRunEnrollmentProps & CommonData
     case step === Step.ENROLLED:
       return (
         <React.Fragment>
-          <a href={courseRun.resource_link} className="course-run-enrollment__cta">
-            <FormattedMessage {...messages.goToCourse} />
-          </a>
+          {new Date().toISOString() >= courseRun.start ? (
```

Comment:
Is this timezone aware?

Reply:
Hmm, good point, I'll check this.
In fact I was using timestamp, but I've seen that ISO strings were used elsewhere.
If you're OK, I can use timestamp instead.